tests/PoliceLogin.yaml

```
@@ -42,4 +42,20 @@ appId: com.anonymous.PoliceMobileApp
 #- tapOn: "Token character 1"
 #- inputText: "ABCDEFGH"
 #- assertVisible: "Access app button"
-#- tapOn: "Access app button"
\ No newline at end of file
+#- tapOn: "Access app button"
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)